schemas/com.snowplowanalytics.snowplow/event/jsonschema/1-0-0

```diff
@@ -0,0 +1,488 @@
+{
+	"$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+	"description": "Schema for a canonical Snowplow event",
+	"self": {
+		"vendor": "com.snowplowanalytics.snowplow",
+		"name": "event",
+		"format": "jsonschema",
+		"version": "1-0-0"
+	},
+	"type": "object",
+	"properties": {
+		"app_id": {
+			"type": ["string", "null"],
+			"maxLength": 255
+		},
+		"platform": {
+			"type": ["string", "null"],
+			"maxLength": 255
+		},
+		"etl_tstamp": {
+			"type": ["string", "null"]
+		},
+		"collector_tstamp": {
+			"type": "string"
+		},
+		"dvce_created_tstamp": {
+			"type": ["string", "null"]
+		},
+		"event": {
+			"type": ["string", "null"],
+			"maxLength": 128
+		},
+		"event_id": {
+			"type": "string",
+			"maxLength": 36
+		},
+		"txn_id": {
+			"type": ["integer", "null"]
+		},
+		"name_tracker": {
+			"type": ["string", "null"],
+			"maxLength": 128
+		},
+		"v_tracker": {
+			"type": ["string", "null"],
+			"maxLength": 100
+		},
+		"v_collector": {
+			"type": "string",
+			"maxLength": 100
+		},
+		"v_etl": {
+			"type": "string",
+			"maxLength": 100
+		},
+		"user_id": {
+			"type": ["string", "null"],
+			"maxLength": 255
+		},
+		"user_ipaddress": {
```

Comment:
I think these "hashable" columns ([link](https://github.com/snowplow/snowplow/commit/4bb64acb0ee5aec8a242338fa34ae3d7e93b6deb#diff-4cff77d747a382e71414088ad196028d)) should be something like `oneOf: [{format: ipv4}, {maxLength: 128}]`